ansible/manila-client.yml

```diff
@@ -4,9 +4,6 @@
 
 # Query OpenStack Manila for details aob
 - hosts: openstack
-  pre_tasks:
```

Comment:
Should `ceph_mount_share_name: HomeDirs` get removed from `ansible/group_vars/all/alaska` too then? I know it won't actually hurt but will be confusing.

Reply:
`ceph_mount_share_name` is used by manila-client.yml further down the file. The debug was accidently commited here last time so therefore removing it.